airflow/requirements.txt

```diff
@@ -1,4 +1,4 @@
-calitp==2022.9.8
+calitp==2022.9.12a0
```

Comment:
can we finalize https://github.com/cal-itp/calitp-py/pull/81, merge it, and do a proper full release to use?